src/nvim/ops.c

```diff
@@ -5732,3 +5738,19 @@ const yankreg_T *op_register_get(const char name)
   }
   return &y_regs[i];
 }
+
+/// Set the previous yank register
+///
+/// @param[in]  name  Register name.
+///
+/// @return true on success, false on failure.
+bool op_register_set_previous(const char name)
```

Comment:
Because it can fail (if `name` is an invalid register name) and for completeness' sake I wanted it to communicate that.